File: Acceso a Datos/Evaluacion_2/WorkSpace2daEvaluacionSpring/demo-6/src/main/java/com/maraloedev/entities/Usuario.java
package com.maraloedev.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@Table(name = "usuario")
@AllArgsConstructor
@NoArgsConstructor
public class Usuario {
	
	@Id
	private long id;
	
	@Column(name = "nombre")
	private String nombre;
	
	@Column(name = "apellidos")
	private String apellidos;

}

File: PMDM/Evaluacion2da/Ejercicios/Kotlin/Ejercicio018_SharedPreferences_SQLite_y_Fragments/app/src/main/java/com/maraloedev/ejercicio018_sharedpreferences_sqlite_y_fragments/Usuario.kt
package com.maraloedev.ejercicio018_sharedpreferences_sqlite_y_fragments

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "usuario")
data class Usuario(

    @PrimaryKey(autoGenerate = true) val id:Int,
    @ColumnInfo(name = "nombre") val nombre:String,
    @ColumnInfo(name = "numero_Telefono") val numero_Telefono:String

)



